src/client/mapblock_mesh.cpp

```diff
@@ -910,12 +917,15 @@ static void updateFastFaceRow(
 			getTileInfo(data, p_next, face_dir,
 					next_makes_face, next_p_corrected,
 					next_face_dir_corrected, next_lights,
+					waving,
 					next_tile);
 
 			if (next_makes_face == makes_face
 					&& next_p_corrected == p_corrected + translate_dir
 					&& next_face_dir_corrected == face_dir_corrected
 					&& memcmp(next_lights, lights, ARRLEN(lights) * sizeof(u16)) == 0
+					// don't tile waving water if we trying to animate it
```

Comment:
I think 'animate' is confusing here as it implies animation of the texture.
Perhaps just
`// don't tile waving water`
is enough?

Reply:
I like that! Will do that.

Reply:
Ideally, with a capital letter 'Don't ...'

Reply:
:) OK.

Reply:
Done